boards/arm/stm32h7/portenta-h7/scripts/flash.ld

```diff
@@ -0,0 +1,210 @@
+/****************************************************************************
+ * boards/arm/stm32h7/portenta-h7/scripts/flash.ld
+ *
+ * Modified from:
+ *
+ * boards/arm/stm32h7/stm32h747i-disco/scripts/flash.ld
+ *
+ *   Copyright (C) 2018 Gregory Nutt. All rights reserved.
+ *   Author: Gregory Nutt <gnutt@nuttx.org>
```

Comment:
could change to apache license

Reply:
@microhobby ditto

Reply:
@antmerlino some files are still missing the fix for the apache license